src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -414,7 +415,10 @@ int enroll_tpm2(struct crypt_device *cd,
                 log_debug_errno(r, "PCR policy hash not yet enrolled, enrolling now.");
         else if (r < 0)
                 return r;
-        else {
+        else if (use_pin) {
+                log_debug("This PCR set is already enrolled, rotating PIN.");
+                *slot_to_wipe = r;
```

Comment:
so we have this coding style rule in our codebase, that function return parameters should:

1. always be called `ret_xyz`
2. always be initialized on success (i.e. if the function returns some success error return, i.e. usually >= 0)
3. never be touched on failure (i.e. if the function returns < 0)

We follow this pretty thoroughly in our casebase.

Hence, here, in the other success paths, please make sure to initialize `*ret_slot_to_wip = -1`, so that this rule is upheld.

Just to minimize surprises.